src/builder.rs

```diff
@@ -645,16 +761,17 @@ impl Builder {
     /// The returned bundle will have no proof or signatures; these can be applied with
     /// [`Bundle::create_proof`] and [`Bundle::apply_signatures`] respectively.
     #[cfg(feature = "circuit")]
-    pub fn build<V: TryFrom<i64>>(
+    pub fn build<V: TryFrom<i64>, FL: OrchardFlavor>(
         self,
         rng: impl RngCore,
-    ) -> Result<Option<(UnauthorizedBundle<V>, BundleMetadata)>, BuildError> {
+    ) -> Result<UnauthorizedBundleWithMetadata<V, FL>, BuildError> {
```

Comment:
_From str4d ([r2602993570](https://github.com/zcash/orchard/pull/471#discussion_r2602993570))_

Per my earlier comment, revert this change:
```suggestion
    ) -> Result<Option<UnauthorizedBundleWithMetadata<V, FL>>, BuildError> {
```

Reply:
TODO